e2e/cypress/integration/orbit-blog.spec.js

```diff
@@ -0,0 +1,22 @@
+describe('ORBIT Blog', () => {
+  before(() => {
+    cy.loginTestIDP()
+  })
+
+  beforeEach(() => {
+    cy.preserveLoginCookies()
+    cy.visit('/')
+  })
+
+  it('can visit the ORBIT Blog page', () => {
+    // TODO: Once we have data in CMS test db, we can test
+    // that articles and pagination are working as expected.
```

Comment:
Praise: good idea on having a place holder with a TODO, should make coming back later to add the test easier to find.